PetShare/appsettings.json

```diff
@@ -16,5 +16,12 @@
   "Imgur": {
     "UploadUrl": "https://api.imgur.com/3/image",
     "ApiKey": "SuperSecretKey"
+  },
+  "Email": {
+    "APIKey": "sendgridApiKey",
+    "TemplateId": "sendGridTemplateId",
+    "SenderEmail": "petshare@yopmail.com",
+    "SenderUsername": "PetShare",
+    "SendEmailsInDevMode": false // <- if true ApiKey and TamplateId must be valid
```

Comment:
`//` comments are not valid in JSON. We can either add a `comment` property below this, or just omit it.